condition/condition_plugins/package.xml

```diff
@@ -9,15 +9,19 @@
 
   <license>TODO</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
 
+  <depend>geometry_msgs</depend>
   <depend>pluginlib</depend>
+  <depend>rclcpp</depend>
   <depend>scenario_conditions</depend>
+  <depend>scenario_intersection</depend>
   <depend>scenario_logger</depend>
   <depend>scenario_logger_msgs</depend>
   <depend>scenario_utility</depend>
 
   <export>
     <scenario_conditions plugin="${prefix}/plugins.xml" />
```

Comment:
What does this export do?

Reply:
I don't really know :) But `scenario_conditions` is the package that loads these plugins (and also provides the base class for them).

Reply:
@fred-apex-ai @nnmm I was not sure if this was doing anything so i removed it in https://github.com/tier4/scenario_runner.iv.universe/pull/19. I assumed `pluginlib_export_plugin_description_file` that is available does the job required here but i am not well informed on this.

Reply:
From http://wiki.ros.org/pluginlib/Tutorials/Writing%20and%20Using%20a%20Simple%20Plugin#Exporting_Plugins, I'd say yes, it's needed. The name looks ok, too

-----

```xml
<export>
 <pluginlib_tutorials_ plugin="${prefix}/polygon_plugins.xml" />
 </export>
```

> The name of the tag, pluginlib_tutorials_ above, should correspond to the package where the base_class for the plugin lives. In this case, the base class and the inherited plugin classes live in the same package, but in most real-world situations this will not be the case. From there, the plugin attribute should be set to point to the XML file generated in the step above.

Reply:
@fred-apex-ai Thanks for checking it out. I brought it back.